PactNet/PactUriOptions.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace PactNet
 {
+    [Obsolete("Please use PactHttpOptions instead")]

```

Comment:
Why the need to introduce `PactHttpOptions` rather than keep using `PactUriOptions`? In my mind it would simplify the change and prevent the breaking changes. It was named `PactUriOptions` in case we needed to add options for consumers reading files from disk rather than via HTTP.

Reply:
You could still add your new fluent methods and add a default constructor.

Reply:
Yep, i wondered about this renaming... My rational is:
 - there are no options related to reading the file localaly yet, so PactHttpOptions is more explicit. 
 - If the need for local file reading hasn't come yet, it may never come, so i prefer having a more explicit type right now than a looser one that might make more sense later. 
 - if ever at some point we need some, i'm not a big fan of bag objects that carry a bunch of settings that you only fill partialy depending on the context, and i'd rather create another class like PactLocalFileOptions with an overload in the main method PactUri(...), and maybe throw ArgumentExceptions if you mismatch the type of Uri with the type of options in arguments.

But if you disagree with that, the renaming is a specific commi and, i can revert it.

Reply:
Yeah I definitely don't disagree with you points, however:

> if ever at some point we need some, i'm not a big fan of bag objects that carry a bunch of settings that you only fill partialy depending on the context, and i'd rather create another class like PactLocalFileOptions with an overload in the main method PactUri(...), and maybe throw ArgumentExceptions if you mismatch the type of Uri with the type of options in arguments.
Yeah me either, however it never starts as a big bag, it slowly creeps up on you. My thoughts at the moment, is that renaming doesn't really make things much better in my eyes, however it does add a bunch of deprecation, which then makes me think about how we go about deprecating it and how long that needs to live for. I feel like maybe let's just add the fluent methods and keep the same name for now.

Reply:
fair enough :)

Reply:
Done